app/client/ui/tooltips.ts

```diff
@@ -240,43 +240,59 @@ export function tooltipCloseButton(ctl: ITooltipControl): HTMLElement {
 }
 
 /**
- * Renders an info icon that shows a tooltip with the specified `content` on click.
+ * Renders an icon that shows a tooltip with the specified `content` on click.
  */
+function iconTooltip(
```

Comment:
I would leave the things here as they were and would just create the iconTooltip helper for the card view.

Reply:
I am creating a helper for the card view but it is a lot of code duplicated for only one icon difference and one style (`display: inline-block`)